include/bluetooth/gatt_dm.h

```diff
@@ -268,17 +268,26 @@ const struct bt_gatt_dm_attr *bt_gatt_dm_desc_next(
  * another one, wait for the result of the previous procedure to finish
  * and call @ref bt_gatt_dm_data_release if it was successful.
  *
+ * If the service UUID is set when you start service discovery using the
+ * @ref bt_gatt_dm_start function. If there are more instances of the same service
+ * on the GATT server, it discovers the next instance. Use this function to discover
+ * all service instances with the given UUID.
```

Comment:
This still does not make sense. The paragraph starts with "If...", but there is no explanation of what follows then.

Reply:
Sorry I supposed that I removed this whole paragraph

Reply:
Done